api-specs/api_spec_PGS.yaml

```diff
@@ -414,11 +414,55 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/XPayAuthResponseEntity500'
-  /request-payments/xpay/auth/{requestId}:
+  /request-payments/xpay/{requestId}/resume:
+    get:
+      summary: payment request to xPay
+      tags:
+        - XPay
+      operationId: resumeXPayPayment
+      parameters:
+        - name: requestId
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: params
+          in: query
+          required: true
+          schema:
+            type: object
+            additionalProperties:
```

Comment:
can you give me an example? is query param with one object or multiple query param?

Reply:
It is a multiple query param. 
Code side is a map of type <String, String>, this is an example of how the API is called via postman
![image](https://user-images.githubusercontent.com/102511515/196970603-c80f9656-5f50-4531-b4aa-2bcb864dfacc.png)

Reply:
why not specify each field? as optional